config/routes.rb

```diff
@@ -1,9 +1,12 @@
 Rails.application.routes.draw do
-  resources :bookings, only: [:create, :show, :destroy, :update]
+  resources :bookings, only: [:create, :show, :destroy, :update] do
+    resources :reviews, only: [:create]
```

Comment:
How does this routing play in with the front end, considering we want to have the review appear on the "gears" show page? Waiting to hear back from Sergio on this.